### PR TITLE
Statements and balances

### PR DESCRIPTION
**Pull request by c-schwarze**

### CODE REVIEW

**Comment by c-schwarze** [comment]
@josiahtillman and @phg49389 can you guys take a look at this?

The logic to hide/show everything got out of hand really fast, but I'm not sure there really is a more clean way to do it.

**Comment by c-schwarze** [comment]
screen shot 2017-08-04 at 3 30 43 pm

File: app/channels/statements_and_balances_channel/statements_and_balances.html [hunk 1]
```diff
@@ -1,45 +1,77 @@
...
-    {% if sodexo_error %}
-        <div>No Dining Services Balance.</div>
-    {% else %}
-        {% if sodexo_balance['mealremain'] %}
-            <div>
-                <p><b>Remaining Meals</b></p>
-                <p>{{ sodexo_balance['mealremain'] }}</p>
-            </div>
+        {# show if there is another section #}
+        {% if (payment_details_check or estimated_billing_check or payment_plans_roles) or (flex_check) %}
```
Comment (phg49389) [inline app/channels/statements_and_balances_channel/statements_and_balances.html]: Why is there a parenthetical 3-OR that's then OR'd with a parenthetical single boolean? Wouldn't it make more sense to just do a 4-OR?

File: app/channels/statements_and_balances_channel/statements_and_balances.html [hunk 2]
```diff
@@ -1,45 +1,77 @@
...
+                {% endif %}
+                {% if payment_plans_check %}
+                    <a href="https://www.bethel.edu/its/blink/touchnet-login/login-tbp">Payment Plans</a>
+                {% endif %}
+            </p>
         </div>
+
+        {# show if flex_roles will be displayed #}
+        {% if flex_check %}
+            <hr />
```
Comment (phg49389) [inline app/channels/statements_and_balances_channel/statements_and_balances.html]: Is more content going to be added to this section at a later date?

Comment (phg49389) [inline app/channels/statements_and_balances_channel/statements_and_balances.html]: Or maybe it should be moved down to the other if flex_check block, below?

File: app/channels/statements_and_balances_channel/statements_and_balances.html (hunk 1 above)
Comment (c-schwarze) [inline app/channels/statements_and_balances_channel/statements_and_balances.html]: The parentheses around the (payment_details_check or estimated_billing_check or payment_plans_roles) boolean is to keep those 3 together. They are essentially a single unit. I thought about maybe setting those to a variable, but I decided against it.

File: app/channels/statements_and_balances_channel/statements_and_balances.html (hunk 2 above)
Comment (c-schwarze) [inline app/channels/statements_and_balances_channel/statements_and_balances.html]: That is all that is displayed. It is just there to make sure we have the `<hr/>`'s in between the sections, if they are applicable to the user.

**Pull request merged**